docs/conductor/getting-started.md

```diff
@@ -0,0 +1,88 @@
+We recommend to create a dedicated authentication token with update access. You can copy n paste the contents for the secret from the Private Packagist UI while creating the token in "Settings" -> "Authentication Tokens". Remove the single quotes around the value.
+
+![Create Authentication Token](/Resources/public/img/docs/conductor/authentication-token.png)
+
+The contents of the variable should look like 
+
+```json
+{"http-basic": {"repo.packagist.com": {"username": "token", "password": "packagist_out_73a81c7eb525b13b6bc22a410b2146e78a38b324f609bf1158c583c704f64cbdd349" }}}
+```
+
+Conductor needs to verify your setup before you can [start receiving Pull Requests](#how-scheduling-works). 
+
+## How scheduling works
+
+- Go to your package on the updates tab in your Private Packagist organization 
+- Click on the name of your package
+
+The list shows groups of all available updates to be scheduled. Each group of updates is called a task. Conductor will schedule only one task at a time. All others are waiting for the task on top of the list to be successful or paused.  
+Once Conductor schedules a task it sends a payload to your CI Platform that triggers the workflow you just added. The payload contains the commands Composer will run to update a group of dependencies from your package.
+
+The workflow consists of several steps:
+
+- code checkout
+- php setup
+- Composer install
+- run Composer update commands from the payload
+- commit changed files (composer.lock, ...)
+- push commit to a new branch
+- Send the status of the workflow to Private Packagist via curl
+
+If the update succeeded, Private Packagist creates a pull request for the just pushed commit. The PR description will contain a reviewable list of all dependencies and parts of their changelogs.  
+When you reviewed the changes and merged the PR, Conductor schedules the next task.    
+If you close the PR, the task will be paused and Conductor schedules the next task. This is the same effect as using the "Pause" button in the UI.  
+If you want to, you can schedule any other task of the list, by clicking on it's name and using the button "Schedule now to create a PR".  
+
+Tasks fixing security issues have a higher priority. They will be moved to the top of the list and scheduled right away.  
+
+## Verify your CI setup
+
+![Task list with verification task](/Resources/public/img/docs/conductor/verification-task-list.png)
+
+Right now all tasks are waiting for the CI verification task on top of the list. Conductor will not start with the regular schedule until this verification task was successful.
+The verification task will only execute `composer update nothing` and will not result in a PR to be sent to your Code Hosting Platform.  
+
+- Click on the task "Verify the continuous integration setup"
+- Use the "Schedule now" button to test your setup
```

Comment:
This implies that they have to manually trigger this but don't we automatically schedule the task?

Reply:
Yeah, two thoughts about this:

1. When we do it like that (not automatically) then we teach them to schedule a (regular) task, too.   And look at the logs of a task.

2. They are then able to see the restart button (which they probably need) right away, and we dont need to navigate them to the task details page.

And yes. Would probably stop doing this automatically. At least not on the button that says "Save"

Reply:
Maybe we need to have an additional step after save, which asks the user to confirm that they have set up the action?

Reply:
Regarding the log/debug. Not sure if the person who set this up will be the same person that reviews/approves PRs and interacts with Conductor on a regular basis. So not sure how beneficial this is.

Reply:
I guess we just have to learn how people set this up.

Reply:
https://trello.com/c/51Mq8cEn/4583-conductor-schedule-the-ci-verification-task-in-a-second-step

Reply:
Feel free to change the docs here, when this changes.

I think we really have to keep this up to date with the latest changes in the UI now